### ISSUE TITLE
nuxtServerInit in server side doesn't have req and res objects

### ISSUE THREAD

**Issue by therealak12** [nuxt/nuxt.js#7863]
### Versions
- node: 14.7.0

### Reproduction
```javascript
export default {
  actions: {
    nuxtServerInit (first, context) {
      if (process.server) {
        console.log(context.req)
      }
    }
  }
}
```

### What is Expected?
On server side I should have access to req and res.

### What is actually happening?
I have the above code in index.js file in store, but `undefined` is logged in the console :(

**Comment by 0721Betty** [comment]
Hello，I encountered the same problem, how did you solve it?

**Comment by danielroe** [comment]
@0721Betty It's likely you're running with `{ target: 'server' }` which intentionally omits those from the server-side context. [Read more here](https://nuxtjs.org/blog/going-full-static/#new-config-option-target).

**Comment by 0721Betty** [comment]
@danielroe Thank you for your answer, I am running with { target: 'static' }. It was solved when I upgraded the version from 2.14.12 to 2.15.2
